doc/whats_new/v1.4.rst

```diff
@@ -371,13 +371,14 @@ Changelog
 :mod:`sklearn.preprocessing`
 ............................
 
-- |MajorFeature| :class:`preprocessing.MinMaxScaler` and
-  :class:`preprocessing.MaxAbsScaler` now
+- |MajorFeature| :class:`preprocessing.MinMaxScaler`, :class:`preprocessing.MaxAbsScaler`
+  and :class:`preprocessing.Normalizer` now
   support the `Array API <https://data-apis.org/array-api/latest/>`_. Array API
   support is considered experimental and might evolve without being subject to
   our usual rolling deprecation cycle policy. See
   :ref:`array_api` for more details.
-  :pr:`26243` by `Tim Head`_ and :pr:`27110` by :user:`Edoardo Abati <EdAbati>`.
+  :pr:`26243` by `Tim Head`_ , :pr:`27110` by :user:`Edoardo Abati <EdAbati>` and
+  :pr:`27558` by :user:`Edoardo Abati <EdAbati>`.
```

Comment:
Same comment in another PR: I am not really in love with stuffing all the estimators into one changelog entry. For me one changelog entry should be about one thing. What do people think about having one entry per estimator?

Reply:
for now I have changed to the format suggested in the PR, not sure if this is exactly what you had in mind :)